src/components/Spinner.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styled, { keyframes } from 'styled-components';
+import { IconLoaderOutline, BaseIconProps } from '@taskany/icons';
+
+const rotate = keyframes`
+    100% {
+        transform: rotate(360deg);
+    }
+`;
+
+const StyledSpinner = styled.div`
+    margin-top: -6px;
```

Comment:
I don't think it's ok. Smth wrong with icon or align?

Reply:
yes, it slipped accidentally 🤦